templates/repo/diff/comments.tmpl

```diff
@@ -1,13 +1,17 @@
 {{range .comments}}
 
 {{ $createdStr:= TimeSinceUnix .CreatedUnix $.root.Lang }}
+{{ $posterName := .Poster.Name }}
```

Comment:
Please move this logic out to user method `GetDisplayName` and use it everywhere instead of duplicating it in all tenplates

Reply:
There's already a  `DisplayName()` function. 
Should I reuse this one and check if the bool is set (which changes behaviour because that func always tries to return Full Name) or create a separate one `GetDisplayName()` like you suggest ?